pkgs/development/python-modules/django-rapyd-modernauth/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+    lib, buildPythonPackage, fetchPypi,
+
+    django,
+    python-dotenv,
+    pytz,
+    setuptools,
+    sqlparse,
+    zipp
+}:
+
+buildPythonPackage rec {
+    pname = "django-rapyd-modernauth";
+    version = "0.0.4";
+
+    src = fetchPypi {
+        inherit pname version;
+        sha256 = "sha256-kDZjI32LcKsmLI38ruINKOUfi6lWTIqXF5qIQHi0LeQ=";
+    };
+
+    meta = with lib; {
+        description = "A Django application that provides a custom User model where the username is the email address.";
+        homepage = "https://github.com/karthicraghupathi/django_rapyd_modernauth";
+        license = licenses.afl20;
+        maintainers = [ maintainers.greg ];
+    };
+
+    doCheck = false;
```

Comment:
Why are tests disabled? We want to run them.

Reply:
Because the tests do not get packaged with the PyPI package by upsteam, so they can't be run during our install.

Reply:
Can we fetch from github instead?

Reply:
Re-attempting with the GH release.